components/Cheese/Cheese.css

```diff
@@ -3,21 +3,52 @@
   flex-direction: row;
   justify-content: space-around;
   flex-wrap: wrap;
+  margin-top: 15px;
 
   @media only screen and (max-width: 650px) {
     flex-direction: column;
   }
 }
 
 .percent {
-  margin-bottom: 10px;
+  position: relative;
+  bottom: -17px;
+  margin-top: -17px;
+  z-index: 1;
 }
 
 .section {
-  width: calc(100% / 3);
-  text-align: center;
+  /* this actually better than calc */
+  width: 33.3333%;
+  display: flex;
```

Comment:
Well calc produce same value (or almost same) as this constant, so there is no reason for calc if we can do is without, wasn't I right?